### PR TITLE
Docs: add issue-specification-document

### LINKED ISSUE #42: Docs: developers issue specification
### Description

**issue specification**
### for contributor
- developers can claim it by comment （want to claim issueId）
- developers should add full labels for issue you raised.(including kinds(类型 such as,feat,bug),Degree of importance(重要程度 emergency，important，normal),strategy(处置策略 such as self ,paticipant,coperation),progress(进度 such as running,later),size(问题规模 such as L,M,XXL))
### for maincontainer
- add projects such as, sprint2023.
- add milestone such as, v0.0.1.
- have to assign one or more who to solve it.
### for commiter
- add labels to each issue and check it out.
- format issue style when needed.
- keep in touch with each contributor.
- Keep up with the status of the issue.
- assign assign one or more who to solve it.

### Screenshots

none

### Additional information

none

### PR DESCRIPTION
**Pull request by AllianceTing**
</br>

#### 🔍 What type of PR is this?

- [✅] My pull request adheres to the code style of this project
- [✅] My code requires changes to the documentation
- [✅] I have updated the documentation as required
- [✅] All the tests have passed

#### 🅰 Which issue(s) this PR fixes:

Fixes #42

### CODE REVIEW

File: docs/ISSUE_SPECIFICATION.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# issue specification
+
+- we use [lot issues] (https://github.com/openIMSDK/openKF/issues) before reporting bugs, Please make sure that has been
+  searching similar [question] (https://github.com/OpenIMSDK/OpenKF/issues), because they may have already answered or
+  is being repaired. New questions should be directed to [Question Assistant](https://openIMSDK/issue-helper?
+  repo=openKF) submitted. For bug reports, include code that can be used to reproduce the problem and how to reproduce
+  it. For new feature suggestions, indicate the changes you want and the expected behavior.
+
+## **for commiter**
+
```
Comment (charles-chenzz) [inline docs/ISSUE_SPECIFICATION.md:10]: typo, should be committer

File: docs/ISSUE_SPECIFICATION.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+- we use [lot issues] (https://github.com/openIMSDK/openKF/issues) before reporting bugs, Please make sure that has been
+  searching similar [question] (https://github.com/OpenIMSDK/OpenKF/issues), because they may have already answered or
+  is being repaired. New questions should be directed to [Question Assistant](https://openIMSDK/issue-helper?
+  repo=openKF) submitted. For bug reports, include code that can be used to reproduce the problem and how to reproduce
+  it. For new feature suggestions, indicate the changes you want and the expected behavior.
+
+## **for commiter**
+
+##### **working for pr/mr**
+
```
Comment (charles-chenzz) [inline docs/ISSUE_SPECIFICATION.md:12]: pr only, this is not gitlab we don't have mr

File: docs/ISSUE_SPECIFICATION.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+## **for contributor**
+
+##### **raise issue**
+
+- developers should add full labels for issue you raised.(including **kind**(问题类型 such as,feat,bug),**Degree of
+  importance**(重要程度 emergency，important，normal),**strategy**(处置策略 such as self ,paticipant,coperation),*
+  *progress**(进度 such as running,later),**size**(问题规模 such as L,M,XXL))
+
+##### **claim issue and meke pr**
+
```
Comment (charles-chenzz) [inline docs/ISSUE_SPECIFICATION.md:34]: typo

File: docs/ISSUE_SPECIFICATION.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
+# issue specification
+
+- we use [lot issues] (https://github.com/openIMSDK/openKF/issues) before reporting bugs, Please make sure that has been
+  searching similar [question] (https://github.com/OpenIMSDK/OpenKF/issues), because they may have already answered or
```
Comment (charles-chenzz) [inline docs/ISSUE_SPECIFICATION.md:4]: the link didn't embed

**Comment by charles-chenzz** [comment]
btw the commit message should not contain info like issues, most likely it will make people confuse
image

File: docs/ISSUE_SPECIFICATION.md (hunk 4 above)
Comment (AllianceTing) [reply docs/ISSUE_SPECIFICATION.md:4]: i will add the link later, before v0.0.2

File: docs/ISSUE_SPECIFICATION.md (hunk 2 above)
Comment (AllianceTing) [reply docs/ISSUE_SPECIFICATION.md:12]: fix it already

File: docs/ISSUE_SPECIFICATION.md (hunk 4 above)
Comment (AllianceTing) [reply docs/ISSUE_SPECIFICATION.md:4]: maybe  document is not compatible， i will check it

Comment (AllianceTing) [reply docs/ISSUE_SPECIFICATION.md]: already fix it

File: docs/ISSUE_SPECIFICATION.md (hunk 3 above)
Comment (AllianceTing) [reply docs/ISSUE_SPECIFICATION.md:33]: fix it already

File: docs/ISSUE_SPECIFICATION.md (hunk 1 above)
Comment (AllianceTing) [reply docs/ISSUE_SPECIFICATION.md:9]: fix it already

**Pull request merged**